src/cascadia/UnitTests_TerminalCore/ConptyRoundtripTests.cpp

```diff
@@ -364,3 +410,208 @@ void ConptyRoundtripTests::WriteAFewSimpleLines()
 
     verifyData(termTb);
 }
+
+void ConptyRoundtripTests::TestResizeHeight()
+{
+    // This test class is _60_ tests to ensure that resizing the terminal works

```

Comment:
OK, well, with this level of coverage... maybe I'm not that mad about the PTY-specific stuff going into `ResizeWithReflow` because if we decide it's a bad place for it later, this test should cover it and make sure we didn't screw up the future refactor.